src/components/question.tsx

```diff
@@ -25,6 +26,9 @@ export default function Question({
   return (
     <Stack spacing={2} {...props}>
       <Text>{`Q${questionIndex + 1}: ${content}`}</Text>
+      {mediaURL && mediaType ? (
```

Comment:
let's do it a bit differently
`!!mediaURL && !!mediaType && <QuestionMedia url={mediaURL} type={mediaType} />`

Reply:
Done